scripts/DocParser.js

```diff
@@ -230,16 +230,18 @@ function init () {
 	const args = parseArgs(process.argv);
 	const strict = args.strict;
 
+	require('./copyGitHub');	// copy GitHub sources
```

Comment:
How would you feel about renaming the "copyGitHub" module to "prepareRaw". "copyGitHub" seems very specific to our current time and set up; it will likely feel really dated in a short while, when the script evolves to do more for us than it does right now.